src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,60 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final Color DEFAULT_COLOR = Color.WHITE;
+    private static final int DEFAULT_RADIUS = 10;
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final Random random = new Random();
+
+    public Figure getRandomFigure() {
+        FigureType type = FigureType.values()[random.nextInt(FigureType.values().length)];
+        Color color = colorSupplier.getRandomColor();
+        switch (type) {
+            case CIRCLE:
+                return getRandomCircle(color);
+            case SQUARE:
+                return getRandomSquare(color);
+            case RECTANGLE:
+                return getRandomRectangle(color);
+            case TRIANGLE:
+                return getRandomRightTriangle(color);
+            default:
+            case TRAPEZOID:
```

Comment:
```suggestion
        Color color = colorSupplier.getRandomColor();
        switch (type) {
            case 0:
                return getRandomCircle(color);
            case 1:
                return getRandomSquare(color);
            case 2:
                return getRandomRectangle(color);
            case 3:
                return getRandomRightTriangle(color);
            default:
            case 4:
```
rather you number, or create constants